chromebook-setup.sh

```diff
@@ -410,7 +452,9 @@ cmd_format_storage()
     wait_for_partitions_to_appear
     find_partitions_by_id
 
+    sudo mkfs.ext4 -L kernel "$CB_SETUP_STORAGE1"
```

Comment:
I am not sure why is needed this change. In fact the kernel partition is not a ext4 partition but a raw partition. Probably is overwritten when the kernel is flashed. Did you try to remove this line?

Reply:
ok, I'll remove it then.